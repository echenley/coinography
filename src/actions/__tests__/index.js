import * as actions from '../index'

describe('actions', () => {
  it('exports correct actions', () => {
    expect(actions).toMatchSnapshot()
  })
})
