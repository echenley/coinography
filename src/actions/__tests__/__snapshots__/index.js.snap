// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions exports correct actions 1`] = `
Object {
  "fetchChartData": [Function],
  "updateChartType": [Function],
  "updateCurrency": [Function],
  "updateGranularity": [Function],
}
`;
